src/components/LanguageSelector/LanguageSelector.module.scss

```diff
@@ -0,0 +1,50 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  width: 100%;
+
+  .label {
+    font-size: 14px;
+    font-weight: 500;
+    white-space: nowrap;
+    color: inherit;
+  }
+
+  .select {
+    width: 100%;
+    height: 40px;
+    padding: 8px 12px;
+    border: 2px solid transparent;
+    border-radius: 10px;
+    font-size: 14px;
+    background-color: #f3f3f3;
+    cursor: pointer;
+    outline: none;
+    transition: all 0.5s;
+
+    &:hover,
+    &:focus {
+      border: 2px solid #4a9dec;
+      box-shadow: 0px 0px 0px 7px rgb(74, 157, 236, 20%);
+      background-color: white;
+    }
+
+    &:disabled {
+      opacity: 0.5;
+      cursor: not-allowed;
+    }
+  }
+
+  // Tablet and desktop
+  @media (min-width: 768px) {
+    flex-direction: row;
+    align-items: center;
+
+    .select {
+      flex: 1;
+      max-width: 300px;
+    }
+  }
```

Comment:
I'd prefer this out of scope and targeting relevant classes within the breakpoint